src/app/recipes/services/delete-recipe.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from "@angular/core";
+import { MatDialog } from "@angular/material/dialog";
+import { EMPTY, Observable, switchMap } from "rxjs";
+import { DialogMessageComponent } from "../../ui/dialog-message/dialog-message.component";
+import { HttpRecipesService } from "./http-recipes.service";
+import { Recipe } from "../models/recipe";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DeleteRecipeService {
```

Comment:
For handling dialogs we usually create something like RecipeDialogService instead of creating separate service for a single action, but for this case I think its fine.